app/common/dispatcher/dispatchAction.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function (name, action) {
```

Comment:
I wonder if it might have been better to just re-use dispatch from appDispatcher but I guess that might be a little confusing.